### ISSUE TITLE
Speed Types not Importing

### ISSUE THREAD

**Issue by roxim5** [TikaelSol/PF2e-Animal-Companions#13]
Reporting here directly since this seems to be the official module Github. I mentioned this in pf2e channel on the Foundry Discord.

Noticed this when dragging the Snake to an empty PC actor. The Climb and Swim speeds didn't get added as expected. I have not tested with other animal companions. This isn't a pressing issue, but just wanted to report it so that it is documented.

**Comment by TikaelSol** [comment]
Yeah, I mentioned in discord that it's a matter of me just missing the speeds since I was trying to untangle the web of other things.  fixing it will be easy when I have a few minutes, though this module is going to get absorbed into core soon hopefully.

**Comment by roxim5** [comment]
Fair, don't duplicate work if you don't need to. Again, not pressing and easy enough to add it in manually.

**Comment by TikaelSol** [comment]
Fixed with version 4.3.2.1
